content/fr/ingress.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Ingress
+status: Completed
+category: technologie
+tags: ["fondamental"]
+---
+
+Un Ingress est un ensemble de règles qui participe à gérer le trafic Internet depuis l'extérieur vers un conteneur ou un groupe de conteneurs s'exécutant dans un cluster.
+Il se compose de deux éléments : la ressource Ingress et le contrôleur Ingress. 
+La ressource Ingress est un fichier de configuration qui coexiste avec les autres fichiers manifestes et permet aux administrateurs de configurer le routage du trafic externe. 
+Le contrôleur Ingress est un serveur web qui met en place le routage du trafic selon la configuration de la ressource Ingress.
```

Comment:
J'ai du mal avec la traduction de : "The ingress controller is the web server technology that actually performs ...."  surtout a cause du terme 'technology'.

Reply:
je pense qu'ici, web server technology  veut dire "nginx mais pas que" . Il y a aussi des ingress a base de proxy genre haproxy ou envoy, qui ne sont pas des serveur web a proprement parler. 
Peut-etre que quelque chose comme:
Le contrôleur Ingress est le service s'appuyant sur des technologies web qui effectue le routage du trafic selon la configuration de la ressource Ingress.
Mais je suis pas convaincu par ma phrase non plus

Reply:
@guillaumebernard84  je préfère ta phrase à la mienne, peut être avec ces modifications :  
_Le contrôleur Ingress est le service qui, en utilisant des technologies web, assure le routage du trafic conformément à la configuration définie dans la ressource Ingress._

Qu'en pensent les autres?

Reply:
Moi ça me paraît très bien en tout cas!